ts/boot/configureStoreAndPersistor.ts

```diff
@@ -187,6 +189,15 @@ const migrations: MigrationManifest = {
         messages: {}
       }
     };
+  },
+
+  // Version 11
```

Comment:
Since we don't persist this information we don't need to make a migration to setup this store section

Reply:
removed within  https://github.com/pagopa/io-app/pull/1584/commits/1f1aa543d55f3ed6839b9d3322ebc1754625e667